Public/index.html

```diff
@@ -44,11 +47,24 @@ <h2>Where will you find yourself?</h2>
       <div class="button-wrapper">
         <button id="search-btn">Take me away!</button>
       </div>
-    </div id="results-box">
-    <div>
-    <div id="result"></div>
-    <div id="chatGPT-results"></div>
-  </div>
+    </div>
+    <div id="results-box">
+      <div id="result"></div>
+      <div id="chatGPT-results"></div>
+    </div>
+    <div id="loading-indicator">Loading...</div>
+    <footer>
+      <p class="footer-text">
```

Comment:
This warning message is absolutely incredible. I had a good chuckle when I noticed you had added it.